variables.tf

```diff
@@ -101,3 +101,23 @@ variable "vm_init_script" {
   type        = "string"
   description = "PowerShell script to initialise the VM"
 }
+variable "domain_name" {
+  type        = "string"
+  description = "Name of the domain to be joinned to"
+}
+variable "OU_path" {
+ type        =  "string"
```

Comment:
there is an extra space after the `=  "string"` and both this line 109 and 110 need one more space indentation